DEVELOPING.md

```diff
@@ -56,14 +56,14 @@ changes since the last tag - if so, this will be a minor, if not, it's a patch.
 (This may not sound like semver - but since our major version is a zero, the
 rule is that minors are breaking changes and patches are new features).
 
-1. Re-generate the API docs: `yarn skydoc`
+1. Re-generate the API docs: `yarn skydoc && (cd internal/karma; yarn skydoc)`
 1. May be necessary if Go code has changed though probably it was already necessary to run this to keep CI green: `bazel run :gazelle`
 1. If we depend on a newer rules_nodejs, update the `check_rules_nodejs_version` in `ts_repositories.bzl`
 1. `git commit -a -m 'Update docs for release'`
 1. `npm config set tag-version-prefix ''`
 1. `npm version minor -m 'rel: %s'` (replace `minor` with `patch` if no breaking changes)
-1. Build npm packages and publish them: `TMP=$(mktemp -d -t bazel-release.XXXXXXX); bazel --output_base=$TMP run //:npm_package.publish && cd internal/karma && bazel --output_base=$TMP run //:npm_package.publish`
-1. `git push && git push --tags`
+1. Build npm packages and publish them: `TMP=$(mktemp -d -t bazel-release.XXXXXXX); bazel --output_base=$TMP run //:npm_package.publish && ( cd internal/karma && bazel --output_base=$TMP run //:npm_package.publish )`
+1. `git push upstream && git push upstream --tags`
```

Comment:
nit: This is assuming upstream. On a clean checkout it would be `origin` by default. Same thing on the rules_nodejs instructions.

Reply:
I'll just add a note that we expect it's named that